misaligned.py

```diff
@@ -4,10 +4,16 @@ def print_color_map():
     minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
     for i, major in enumerate(major_colors):
         for j, minor in enumerate(minor_colors):
-            print(f'{i * 5 + j} | {major} | {minor}')
+            global index0
+            pair_number = i * 5 + j

```

Comment:
The 5 in this expression is same from the code given for assignment. It is to calculate the pair number of major, minor color set.

Reply:
What I wanted to emphasize was, the number 5 would lead to confusion or error in future when someone else is maintaining the code.
Also, will this code work of major color or minor colour length is other than 5. Please think over it and write test cases accordingly 😊👍